src/vs/workbench/services/themes/browser/workbenchThemeService.ts

```diff
@@ -793,13 +794,21 @@ class ThemeFileWatcher {
 }
 
 function _applyRules(styleSheetContent: string, rulesClassName: string) {
-	const themeStyles = mainWindow.document.head.getElementsByClassName(rulesClassName);
-	if (themeStyles.length === 0) {
-		const elStyle = createStyleSheet();
-		elStyle.className = rulesClassName;
-		elStyle.textContent = styleSheetContent;
-	} else {
-		(<HTMLStyleElement>themeStyles[0]).textContent = styleSheetContent;
+	let themeStyle = mainWindow.document.head.getElementsByClassName(rulesClassName).item(0);
+	if (!themeStyle) {
+		themeStyle = createStyleSheet();
+		themeStyle.className = rulesClassName;
+	}
+
+	if (themeStyle.textContent !== styleSheetContent) {
```

Comment:
@bpasero Should we put that code in a `applyStyleSheetContent` utility in `src/vs/base/browser/domStylesheets.ts`, so wthat ''data-version'' is an implementation detauil there?
Also I wonder if there are other places where we set the stylesheet content?

Reply:
@aeschli I can clean this up after the PR landed, its quite hard for me currently to push changes into the remote branch.